integration-tests/testsuite/src/it/resources/README.md

```diff
@@ -0,0 +1,38 @@
+# CtrlC.exe
```

Comment:
I think there should be a link from the main README.md to this file.

Reply:
@zakkak Why do you think it is important for the main README? The only reason that comes to my mind would be to explain as early as possible that that is a strange .exe file in the repo so as people don't freak out about it prematurely.   
Otherwise it is an implementation detail (although it took me quite some time to figure out back in the day :see_no_evil: )

Reply:
Yes, if I am going to run this on windows I want to know that a strange binary that I didn't build is going to run on my machine. Ideally I would prefer to build it from source code, not sure how hard that would be though.

Reply:
@zakkak I built that binary (I did not download a random binary and put it in the repo :) ) and it is trivial. Given the fact that running this TS on WIndows without VS doesn't make sense, people are going to have the toolchain installed anyway. I can build it on the fly.     
It would just need to be done at the right time. Lemme think about it. Having any extra scripts defeats the purpose of having Maven there in the first place, so I would probably create some aux module to be built before testsuite module starts....

Reply:
Anyway, I expect having more small native utilities there over the time...

Reply:
> @zakkak I built that binary (I did not download a random binary and put it in the repo :) ) 

I agree, but anyone else who will download the testsuite and run it will essential _download a random binary and run it_ so they might not feel good about it.

I see how solving this is not that trivial, especially with proper maven integration. What if you build it through the test harness (like you invoke native-image)?